File: textAdventure/src/Weapon.java

class Weapon {
    private String name;
    private double accuracy;
    private int dmg;

    public Weapon(String name, double accuracy, int dmg) {
        this.name = name;
        this.accuracy = accuracy;
        this.dmg = dmg;
    }

    public double getAccuracy() {
        return this.accuracy;
    }

    public int getDmg() {
        return this.dmg;
    }

    public String getName() {
        return this.name;
    }
}

File: weapon.py
class Weapon:
    def __init__(self, name, accuracy, dmg):
        self.name = name
        self.accuracy = accuracy
        self.dmg = dmg

    def getAccuracy(self):
        return self.accuracy
    
    def getDmg(self):
        return self.dmg
    
    def getName(self):
        return self.name



